src/components/TableWithBrowserPagination/index.js

```diff
@@ -4,27 +4,10 @@ import classnames from 'classnames';
 import Pagination from '../Pagination';
```

Comment:
fix bug when data change dynamically and then the activePage is greater than data length